rootfs/scheduler/__init__.py

```diff
@@ -685,18 +697,18 @@ def resolve_state(self, pod):
         pod_state = pod['status']['phase']
         return states.get(pod_state, pod_state)
 
+    @retry(stop_max_attempt_number=3, wait_fixed=1000)
```

Comment:
Will it bubble up the exception after the 3rd failed attempt? How is that handled